src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1875,6 +1875,46 @@
     /**
      * The {@code native} keyword.
      *
+     * <p>For example:</p>
+     * <pre>
+     * native void equal(int a, int b)
```

Comment:
`native void foo(){}`

Focus on modifier, all other is not relevant.

Reply:
@romani I've made the requested changes, please review